### PR TITLE
Migrated travis CI build process to github actions

### PR DESCRIPTION
**Pull request by CluEleSsUK**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: build
+on:
+  pull_request:
+    branches: [master]
```
Comment (AnomalRoil) [inline .github/workflows/build.yml:4]: Why only on pull_requests?

I'd expect something like:
```
on:
  push:
    branches: [master]
    tags:
    pull_request:
```

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```
Comment (AnomalRoil) [inline .gitignore:2]: Could you maybe add this to your own global gitignore instead of adding it to all projects?

File: README.md [hunk 3]
```diff
@@ -3,7 +3,6 @@
 <p align="center"><img src="logo.png" width="220" /></p>
 
 <p align="center">
-  <a href="https://travis-ci.org/drand/drand-client" title="Build Status"><img src="https://travis-ci.org/drand/drand-client.svg?branch=master" /></a>
```
Comment (AnomalRoil) [inline README.md:6]: Good catch, I need to do that on the kyber-bls repo too xD

File: .gitignore (hunk 2 above)
Comment (CluEleSsUK) [reply .gitignore]: TIL there is a global gitignore!! updated

File: .github/workflows/build.yml (hunk 1 above)
Comment (CluEleSsUK) [reply .github/workflows/build.yml:4]: was gonna do a separate build + deploy for master in another action once I've got the npm creds etc, but I could potentially make this 
```
on:
  pull_request:
  push:
    branches: master
```

and have a separate deploy only workflow

Comment (CluEleSsUK) [reply .github/workflows/build.yml:4]: + tags

**Review by AnomalRoil — APPROVED** [review]

**Pull request merged**